src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -233,6 +257,27 @@ def launch_fluent(
         raise GPUSolverSupportError()
     _process_kwargs(kwargs)
     del kwargs
+    if show_gui is not None:
+        warnings.warn(
+            "show_gui is deprecated, use exposure instead", PyFluentDeprecationWarning
+        )
+    if show_gui or os.getenv("PYFLUENT_SHOW_SERVER_GUI") == 1:
+        exposure = FluentExposure.GUI
+    del show_gui
+    if exposure is None:
+        # Not using NO_GUI in windows as it opens a new cmd or
+        # shows Fluent output in the current cmd if start is not used
+        exposure = FluentExposure.HIDDEN_GUI if _is_windows() else FluentExposure.NO_GUI
+    if isinstance(exposure, str):
+        exposure = FluentExposure(exposure)
+    if graphics_driver is None:
+        graphics_driver = "auto"
+    graphics_driver = str(graphics_driver)
```

Comment:
Converting possible enum value to string to catch thigs like specifying `FluentLinuxGraphicsDriver` in windows during the enum constructor in the following line.